website/docs/docs/collaborate/git/connect-github.md

```diff
@@ -25,7 +25,9 @@ If you're using an On-Premises GitHub deployment, this method will not work for
 
 ## Installing dbt Cloud in your GitHub account
 
-A GitHub organization owner needs to connect and configure the dbt Cloud app for their GitHub organization. If you are a team of one or wish to connect your personal GitHub account, then these instructions also apply to you.
+:::info Org Admin Permissions Required
+If you are integrating with a GitHub environment that uses Organizations, the following steps must be done by a GitHub org admin who has logged into dbt Cloud with Owner or Account Admin permissions.  This will mean you need to provision an additional dbt Cloud account for your GitHub org admin until these steps have been completed.
```

Comment:
```suggestion
To integrate a GitHub environment that uses Organizations, a GitHub org admin must log into dbt Cloud with _Owner_ or _Account Admin_ [permissions](/docs/collaborate/manage-access/self-service-permissions) and perform the following steps. 
```

Reply:
The note about needing to be an org admin feels like the most important thing in this entire section and is the thing folks seem to regularly miss so I wanted to make that super clear.  That was the motivation for the callout.

However we can do that I am in favor of -- ultimately I don't want folks to miss this critical piece since if you are not an org admin things just silently fail and you get error messages in other places.

That last sentence didn't feel like it added value or clarified things and personal login instructions are detailed toward the bottom so I felt like omitting that sentence gave more strength to what remained.

Reply:
totally makes sense, i was talking to @runleonarun about this and wondered if a H3 Prerequisites header would work @barberscott ?

Reply:
That works for me!